docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -189,10 +189,10 @@ have workloads split across different networks or clouds.
 </Notice>
 
 Using the ports in Teleport's default configuration, the Node needs to be able
-to talk to ports `3080` and `3024` on the Proxy Service. Port `3080` is used to
-initially fetch the credentials (SSH and TLS certificates) and for discovering
-the reverse tunnel. Port `3024 `is used to establish a connection to the Auth
-Service through the Proxy.
+to talk to port `3024` on the Proxy Service, which is used to both fetch the
+credentials (SSH and TLS certificates) and establish a connection to the Auth
+Service through the Proxy. Alternatively, port `3080` can be used to fetch
```

Comment:
Also, if there's no difference between using port 3024 and 3080, we could probably remove the last sentence of this paragraph and say, "...the Node needs to be able to talk to either port `3024` or `3080` on the Proxy Service...".

Reply:
3024 is used in both cases to establish the reverse tunnel connection; it's just a question of whether the node fetches credentials over 3024 or 3080. It would have to be something like "either port `3024` or `3024` AND `3080`", and I'm not sure how to word that elegantly.

Reply:
How does the new behavior work with TLS multiplexing? This paragraph (and the ones after it) seem to assume that TLS multiplexing is disabled by default, while in all supported Teleport versions now (v8 and above), this is not the case.

In any case, I think this paragraph would be much clearer by:

- Replacing specific port numbers with references to Proxy Service configuration settings. Otherwise, the text gets a bit muddled, since we have to go back and forth between port numbers and their usage. If readers need to know the exact ports to use, we could include a command to hit the `/webapi/ping` endpoint.

- Rephrasing "can be used" in the last sentence of the paragraph to something more specific. I'm not sure if 3080 only needs to be _open_ on the Proxy Service or if there's some specific config setting/flag/env variable the user needs to change from something else to 3080.